api/query/query.go

```diff
@@ -14,9 +14,23 @@ import (
 	"github.com/web-platform-tests/wpt.fyi/shared"
 )
 
-// summary is the golang type for the JSON format in pass/total summary files.
-type summary map[string][]int
+// SummaryResult is the format of the data from summary files generated with the newest aggregation method.
+type SummaryResult struct {
+	// Status represents the 1-2 character abbreviation for the status of the test.
+	Status string `json:"s"`
+	// Counts represents the subtest counts (passes and total).
+	Counts []int `json:"c"`
+}
 
+// summary is the golang type for the JSON format in pass/total summary files.
+// It has an old structure and a new structure - each which represent summary files
+// that match the old or new summary format.
+type summary struct {
```

Comment:
In this case, `summary` is an unexported variable but with exported fields `Old` and `New`. Let us make it consistently and make it an exported variable

Reply:
That's true, and I don't even think the export is required here (it was not previously). I'll  change them to no longer export.